libraries/botbuilder-core/tests/ActivityHandler.test.js

```diff
@@ -248,6 +248,19 @@ describe('ActivityHandler', function() {
 
         processActivity({type: ActivityTypes.InstallationUpdate, action: 'add'}, bot, done);
     });
+    
+    it(`should fire onInstallationUpdateAddUpgrade`, async function(done) {
```

Comment:
You can't do both `async` and `done` style testing with mocha. Prefer just the `async` pattern.

Reply:
Good catch. It used to show warnings for that. I'll clean that up in all the tests in this file...there's a lot of them.